README.md

```diff
@@ -20,16 +22,23 @@ A capstone project for UChicago's CAPP122 where abortion related data is taken f
 - `make visualize`: Takes the data produced from the `parse-data` layer and creates the project's visualizations
 
 ## How do we run this thing?
-For the `make visualize` command, you must have called the two commands that are referenced before it for it to run successfully.
-- Without the data from `make api` and `make parse-data`, `make visualize` has nothing to act on.
+There are two ways that you can run this application, one of them is to run all components of it at once and the other is to run each component individually. I will give you the instructions for both methods below.
 
-1. After you have installed [Poetry](https://python-poetry.org/docs/basic-usage/), run the command: `poetry shell`
+#### Run with one command
+1. After you have installed [Poetry](https://python-poetry.org/docs/basic-usage/), run the command from the base repository directory: `poetry shell`
+2. Run the command `poetry install` to install the package dependencies within the project.
+3. Run the `make run` command, and you will run the `api`, `data-parsing`, and `visualization` layers in serial. This will spin up a web page you can access via the URL `localhost:8005`.
+
+#### Run each package individually
+1. After you have installed [Poetry](https://python-poetry.org/docs/basic-usage/), run the command from the base repository directory: `poetry shell`
 2. Run the command `poetry install` to install the package dependencies within the project.
 3. Run the `make api` command to get the data from the _Abortion Policy API_.
 4. Run the `make parse-data` command to parse the data so that we can have our data fit the format needed for the next step.
-5. Run the `make visualize` command to start the [Flask](https://flask.palletsprojects.com/en/2.2.x/) server so that we can visualize the data we have pulled and parsed!
+5. Run the `make visualize` command to start the [Flask](https://flask.palletsprojects.com/en/2.2.x/) server, accessible via the URL `localhost:8005`, so that we can visualize the data we have pulled and parsed!
+   - For the `make visualize` command, you must have called the two commands that are referenced before it for it to run successfully.
+     - Without the data from `make api` and `make parse-data`, `make visualize` has nothing to act on.
 
-### We would like to thank the following organizations for providing our reference data:
+## We would like to thank the following organizations for providing our reference data:
 - _Abortion Policy API_: https://www.abortionpolicyapi.com/
 - _ANSIRH Abortion Facility Database_: https://abortionfacilitydatabase-ucsf.hub.arcgis.com/
 - _i need an a_: https://www.ineedana.com/
```

Comment:
add attribution for the geo json data!

Reply:
That one is one the viz branch! Merge that boi and I'll put it in!